akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtoAnySerialization.scala

```diff
@@ -215,7 +278,20 @@ import scalapb.options.Scalapb
         .getMethod("parser")
         .invoke(null)
         .asInstanceOf[Parser[com.google.protobuf.Message]]
-      Some(new JavaPbResolvedType(parser))
+      val akkaSerializer = Try(serialization.serializerFor(clazz)).toOption
+        .flatMap {
+          case _: DisabledJavaSerializer =>
+            // There will be a warning log from Akka "Using the Java serializer", which need additional clarification
+            // in this context.
+            log.warn(
+              "Missing Akka serializer for protobuf message class [{}]. Consider defining it in " +
+              "`akka.actor.serialization-bindings`. It was bound to disabled Java serializer, which " +
+              "is not what you want.",
+              clazz.getName)
```

Comment:
maybe I should remove the akkaSerializer from the ResolvedType again, and lookup when needed, since it's not often used, and this warning might be confusing

Reply:
> since it's not often used

Isn't it used for every message if you look at the payload in pushed events?

Reply:
yes, but only for producer push. ProtoAnySerialization is used in many more places.

Reply:
Ok, let's drop it then

Reply:
done https://github.com/akka/akka-projection/pull/966/commits/c672ef6cb8ff47ea358dfbb125c33ae60e611bae